doc/enterprise-shield/c_enterprise_shield_use_cases.md

```diff
@@ -0,0 +1,78 @@
+-   [Home](../../index.md)
+-   [Documentation](../index.md)
+-   Security with KAAZING Gateway
+
+Common Enterprise Shield™ Use Cases ![This feature is available in KAAZING Gateway - Enterprise Edition](../images/enterprise-feature.png)
+=====================================
+
+This topic provides a high-level overview of some common Enterprise Shield™ use cases.
+
+Before You Begin
+--------------------------------------
+
+This procedure is part of [Configure Enterprise Shield™ for KAAZING Gateway](o_enterprise_shield_checklist.md):
+
+1.  [Common KAAZING Gateway Production Topologies](../admin-reference/c_topologies.md#enterprise-shield--)
+2.  [About Enterprise Shield™](o_enterprise_shield_checklist.md#about-enterprise-shield)
+3.  **Common Enterprise Shield™ Use Cases**
+4.  [Setting Up the Gateway](../about/setup-guide.md)
+5.  [Walkthrough: Configure Enterprise Shield™](p_enterprise_shield_config.md)
+6.  [Walkthrough: Configure Enterprise Shield™ for High Availability](p_enterprise_shield_cluster.md)
+
+The use case configurations described in this topic vary depending on the level of security, such as if the Gateways are clustered for high availability, whether network transports are used, and so on. Also, the sample topologies use two Gateway services instead of just one to demonstrate the Enterprise Shield™ reverse connectivity behavior in a configuration with multiple services.
```

Comment:
"... such as if the Gateways are clustered for high availability..."

This is not related to security, as the first part of the sentence says.

Not sure what "... whether network transports are used ..." means here.

Perhaps the first sentence can be modified:

> The use case configurations described in this topic vary depending the level of security and high availability you want to configure.